src/linearmixedmodel.jl

```diff
@@ -350,7 +350,10 @@ Return the lower Cholesky factor for the fixed-effects parameters, as an `LowerT
 `p × p` matrix.
 """
 function feL(m::LinearMixedModel)
-    LowerTriangular(m.L[kp1choose2(m.dims.nretrms + 1)])
+    XyL = m.L[end]
+    k = size(XyL, 1)
+    inds = Base.OneTo(k - 1)
+    LowerTriangular(view(XyL, inds, inds))
```

Comment:
more general question (i.e. not a change for this PR): I wonder if we should better document when our accessor methods are returning views / shallow copies and when they're returning deep copies.